packages/css/src/components/image-slider/image-slider.scss

```
@@ -45,13 +45,15 @@
 }
 
 .ams-image-slider__controls {
-  display: flex;
+  display: none;
   grid-column: 1/-1;
   grid-row: 1;
   justify-content: space-between;
 
-  @media (pointer: coarse) and (max-width: $ams-breakpoint-medium) {
-    display: none;
+  // We show controls on non-touch devices, where people can’t swipe the slides,
+  // but not in narrow windows, where they obscure too much of an image.
+  @media (min-width: $ams-breakpoint-medium) and (pointer: fine) {
+    display: flex;
   }
```

I updated the suggestion in my comment a bit later, but apparently you’d already copied the original one. Once more:

```suggestion
  // We hide controls on touch devices, where people can swipe the slides,
  // and also in narrow windows, where they obscure too much of an image.
  @media (min-width: $ams-breakpoint-medium) and (not (pointer: coarse)) {
    display: flex;
  }
```

---

Ah, interesting. I'll try it out.

---

Tested it: in the device simulation in Chromium's devtools the buttons are not visible on any hybrid devices such as the Microsoft Surface devices and the Asus Zendbook Fold.